WorkloadManagementSystem/Agent/JobAgent.py

```diff
@@ -292,7 +295,7 @@ def execute(self):
       self.log.warn('Job has no CPU requirement defined in JDL parameters')
 
     # Job requirement for a number of processors
-    processors = int(params.get('NumberOfProcessors', 1))
+    processors = int(params.get('NumberOfJobProcessors', 1))
```

Comment:
May be this option should be called JobProcessorsRequirement in order not to confuse with a similar name but different meaning NumberOfProcessors option